src/runtime/composables/refresh/useAuth.ts

```diff
@@ -168,7 +172,7 @@ const signOut: SignOutFunc = async (signOutOptions) => {
         | 'options'
         | 'trace';
     }
-    res = await _fetch(nuxt, path, { method, headers })
+    res = await _fetch(nuxt, path, { method, headers, body: method.toLowerCase() === 'post' ? body : undefined })
```

Comment:
May be better to build body only when the check is truthy:
```ts
const body = method.toLowerCase() === 'get'
  ? undefined
  : objectFromJsonPointer(config.refreshToken.refreshRequestTokenPointer, refreshToken.value)
```

Reply:
@phoenix-ru, as this PR is bascilly ready-to-go and has been stale for a long time, I would go ahead and merge this PR! We can optimize this in a follow-up PR!